### PR TITLE
EZP-27323: Setting up Amazon AWS S3 clustering

### PR DESCRIPTION
**Pull request by DominikaK**
Belated PR to https://jira.ez.no/browse/EZP-27323

This should be merged to *1.7*, *1.10* and onwards.
**Important**: when merging to 1.10 and later, the mention of 1.7.4 at the end of the cookbook page must be removed.

### CODE REVIEW

File: docs/cookbook/setting-up-amazon-aws-s3-clustering.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+            io:
+                binarydata_handler: aws_s3
+```
+
+Clear all caches and reload, and that's it.
+
+## Migrate your existing binary data to S3
+
+Since 1.7.4 you can migrate existing binary data to S3 using the `php app/console ezplatform:io:migrate-files` command
+which was added in [EZP-25946](https://jira.ez.no/browse/EZP-25946).
```
Comment (glye) [inline docs/cookbook/setting-up-amazon-aws-s3-clustering.md]: This should link to a page describing the command, but that's a job for me :)

**Comment by DominikaK** [comment]
Thanks, @glye, added a note about experimental status.

**Comment by andrerom** [comment]
This should be adapted for #61:
- S3 is no longer called experimental
- [tip section added there for S3 and Google Storage](https://github.com/ezsystems/developer-documentation/pull/61/files#diff-db783521be2f9624acf428bd1174fb7cR108) can get a link to this doc page

File: docs/cookbook/setting-up-amazon-aws-s3-clustering.md (hunk 1 above)
Comment (glye) [reply docs/cookbook/setting-up-amazon-aws-s3-clustering.md]: This page exists now, at https://doc.ezplatform.com/en/1.11/guide/clustering/#migration

**Comment by DominikaK** [comment]
Thanks, @andrerom and @glye, fixed according to your comments.

**Comment by andrerom** [comment]
@DominikaK Maybe add introduction part after header introducing this as a way to setup cluster _(link to cluster doc) _using S3 binary handler, meaning binary files are stored in AWS S3.  And emphasis reader should probably read cluster doc first to see the big picture of things.

**Comment by DominikaK** [comment]
@andrerom Introduction added per your comment.

File: docs/cookbook/setting_up_amazon_aws_s3_clustering.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
+# Setting up Amazon AWS S3 clustering
+
+When setting up clustering, you can use Amazon AWS S3 as a binary handler,
+meaning it will be used to store binary files.
```
Comment (andrerom) [inline docs/cookbook/setting_up_amazon_aws_s3_clustering.md]: it => AWS S3

To distinguish between flysystem aws s3 handler and the AWS S3 service itself :P

**Pull request merged**